htdocs/expedition/card.php

```diff
@@ -2300,6 +2339,7 @@
 				$product_static->surface_units = $lines[$i]->surface_units;
 				$product_static->volume = $lines[$i]->volume;
 				$product_static->volume_units = $lines[$i]->volume_units;
+				$product_static->stockable_product = $lines[$i]->stockable_product;
```

Comment:
Hi @thomas-Ngr 
I believe this line does not work as $lines[$i]->stockable_product does not exist.